### PR TITLE
Issue CORE-328 | Fix misspells on the repository

### PR DESCRIPTION
**Pull request by brenordr**
# Description

Kind: Fix

Section: All Code

Summary:

Run misspell and fix typos over the repo

Issue link:

## Changelog

> List the main points of change

Touches multiple files, but nothing serious.

## Pay attention to

Only one struct field changed and can be a problem.

> Important parts of the code that require special attention

### CODE REVIEW

File: examples/pubsub/discord/main.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ type discordMessage struct {
 	AvatarURL string `json:"avatar_url"`
 	TTS       bool   `json:"tts"`
 	File      []byte `json:"file"`
-	Embeded   []byte `json:"embeds"`
+	Embedded   []byte `json:"embeds"`
```
Comment (brenordr) [inline examples/pubsub/discord/main.go:20]: This struct changed the field name. Doesn't seem to impact the code.
But worth to check in detail

File: examples/pubsub/discord/main.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ type discordMessage struct {
...
 }
 
 var webhook = "https://discord.com/api/webhooks/823903452475162666/o9aKLMVOb9-ZhfDD7RJ84OCW6WgU2PQnsEj0CzPgFzKC1icqgqWqF8LZxHSFXEzH1NED"
```
Comment (brenordr) [inline examples/pubsub/discord/main.go:23]: @inspr/core This should be a secret. 😅

File: examples/pubsub/discord/main.go (hunk 1 above)
Comment (ptcar2009) [reply examples/pubsub/discord/main.go:20]: is not used currently, only text is sent right now

File: examples/pubsub/discord/main.go (hunk 2 above)
Comment (ptcar2009) [reply examples/pubsub/discord/main.go:23]: yeah, will change, get it on the issues

**Review by ptcar2009 — APPROVED** [review]

File: docs/dapp_overview.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ The overall communication in between Nodes can be seen as going up or down the t
 ![node-communication](img/node-comm.jpg)
 
 ## What are Aliases?
-As shown in the image above, Nodes communicate with each other throught it's parent dApp Channels. This implies that the communication is one-layered only, because the parent dApp, by definition, isn't a Node and therefore isn't able to send or receive messages (it only defines where it's child Nodes can read and write from):
+As shown in the image above, Nodes communicate with each other thought it's parent dApp Channels. This implies that the communication is one-layered only, because the parent dApp, by definition, isn't a Node and therefore isn't able to send or receive messages (it only defines where it's child Nodes can read and write from):
```
Comment (nicholaspcr) [inline docs/dapp_overview.md]: wouldn't it be `through its`? thought is an idea or opinion produced by thinking, or occurring suddenly in the mind.

Comment (ptcar2009) [reply docs/dapp_overview.md]: that's correct

Comment (brenordr) [reply docs/dapp_overview.md]: @ptcar2009 not sure if mean @nicholaspcr was correct. or the text.
I believe he is right. Should be `through its`

**Review by franciscobonand — APPROVED** [review]

**Pull request merged**